source/server/http/admin.cc

```diff
@@ -1102,7 +1102,12 @@ Http::Code AdminImpl::handlerRuntimeModify(absl::string_view url, Http::HeaderMa
   }
   std::unordered_map<std::string, std::string> overrides;
   overrides.insert(params.begin(), params.end());
-  server_.runtime().mergeValues(overrides);
+  try {
+    server_.runtime().mergeValues(overrides);
+  } catch (const EnvoyException& e) {
+    response.add(e.what());
+    return Http::Code::ServiceUnavailable;
```

Comment:
I would probably return a 400 here, though I don't feel very strongly about it.

Reply:
My thinking here is that 503 better reflects the fact that there is a service, but it's not available due to missing config. But I'm also shruggie on it :)

Reply:
@jmarantz probably has a stronger opinion here so I might briefly ping him and see if he cares. I'm fine either way.

Reply:
Happy to continue discussing, but I'll take the feedback in a separate PR if that's OK and merge this one, as this is blocking some other PRs that are time sensitive. I'd like to land the TDS API proto PR that follows this so we can unblock other dependent projects that are waiting for this.

Reply:
:+1: